doc/testplans/pbr_terrain_paintmap.md

```diff
@@ -0,0 +1,48 @@
+# PBR Terrain Paintmap
```

Comment:
Consult this document if you're interested in seeing what the paintmap looks like. The current default is 2k "RGB5".